pkgs/by-name/ss/sscep/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gitUpdater,
+  openssl,
+  autoconf,
+  automake,
+  libtool,
+  pkg-config,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sscep";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "certnanny";
+    repo = "sscep";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wlxQONOCLPuNdI6AyMJoLP09cs+ak7Jv9idhXTT5RWA=";
+  };
+
+  preConfigure = ''
+    ./bootstrap.sh
+  '';
+
+  buildInputs = [ openssl ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+    pkg-config
+  ];
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
```

```suggestion
  passthru.updateScript = nix-update-script { };
```

Should have a smaller footprint